test/integration/hash-sequence.int.c

```diff
@@ -123,33 +123,24 @@ int
 test_invoke (TSS2_SYS_CONTEXT *sapi_context)
 {
     int i;
-
     TSS2_RC rval;
-    TPM2B_AUTH auth = {
-        .size = 2,
-        .buffer = { 0x00, 0xff },
-    };
-
     TPMI_DH_OBJECT  sequenceHandle[MAX_TEST_SEQUENCES];
     TPM2B_MAX_BUFFER dataToHash;
     TPM2B_DIGEST result;
     TPMT_TK_HASHCHECK validation;
-
-    TPMS_AUTH_COMMAND auth_command = {
-        .sessionHandle = TPM2_RS_PW,
-        .hmac = auth,
-    };
-    TPMS_AUTH_COMMAND *auth_command_array[1] = { &auth_command, };
-    TSS2_SYS_CMD_AUTHS cmd_auths = {
-        .cmdAuthsCount = 1,
-        .cmdAuths      = auth_command_array,
+    TPM2B_AUTH auth = {
+        .size = 2,
+        .buffer = { 0x00, 0xff },
     };
-    TPMS_AUTH_RESPONSE  auth_response = { 0 };
-    TPMS_AUTH_RESPONSE *auth_response_array[1] = { &auth_response };
-    TSS2_SYS_RSP_AUTHS  rsp_auths = {
-        .rspAuths      = auth_response_array,
-        .rspAuthsCount = 1
+
+    TSS2L_SYS_AUTH_COMMAND cmd_auths = {
+        .count = 1,
+        .auths = {{
+            .sessionHandle = TPM2_RS_PW,
+            .hmac = auth,
+        }}
     };
+    TSS2L_SYS_AUTH_RESPONSE  rsp_auths = { .count = 1 };
```

Comment:
You actually do not need to set .count anymore.
Recommend `TSS2L_SYS_AUTH_RESPONSE  rsp_auths;` since otherwise static code analysis would complain about unused values.

Reply:
Right, it will be set in Tss2_Sys_GetRspAuths() when we will receive the response.